pages/index.js

```diff
@@ -1,25 +1,47 @@
+import { useEffect, useState } from "react";
 import styled from "styled-components";
-import FoodCard from "../components/FoodCard";
-import useSWR from "swr";
-import { fetcher } from "../helpers/api";
-
-export default function Home() {
-  const { data: recipes, error } = useSWR("/api/recipes", fetcher);
-
-  if (error) return <h1>There was an error</h1>;
-
-  if (!recipes) return <h1>your tasty recipes loading ...</h1>;
+import ChooseFoodCard from "../components/ChooseFoodCard";
 
+function Home({ nextRecipe, onAddRecipesToFavourites, onFindRandomRecipe }) {
   return (
     <main>
       <h1>WTF? What to food </h1>
-      <h2>my super tasty selection of recipes:</h2>
+      <h2>what are you up for?</h2>
+
+      <ChooseFoodCard
+        recipe={nextRecipe}
+        onAddRecipesToFavourites={onAddRecipesToFavourites}
+        onNextRecipe={onFindRandomRecipe}
+      />
 
-      <ul>
-        {recipes?.map((recipe) => (
-          <FoodCard recipe={recipe} key={recipe.id} />
-        ))}
-      </ul>
+      <SytledNav>
+        <StyledImgBox href="/">
+          <img src="/icons/iconHome.png" />
```

Comment:
Each `img` should have an `alt` attribute. You could also use the `Image` component here.